baseplate/ratelimit/__init__.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
```

Comment:
does this work on py2? https://github.com/reddit/baseplate/blob/master/baseplate/__init__.py#L4

Reply:
I don't understand why this wouldn't this work. The tests pass in travis. Is there something more specific you'd like me to look into?

Reply:
hm, i can't repro it but it used to fail. maybe something changed in the minor versions of 2.7. :thinking: